runFfmpeg.sh

```diff
@@ -16,12 +16,12 @@ else
   STREAM_TEXT_PATH=$2
 
   # Souce our config
-  source radioFiles/config.sh
+  .radioFiles/config.sh
 
   # Define our private variables
   # KEYINT represents the number of key frames.
   # Should be somewhere between 2 to 4.
-  KEYINT=$(expr $VIDEO_FPS \* 3)
+  KEYINT=${#VIDEO_FPS \* 30}
```

Comment:
Only thing I can say is this one line.

I don't remember what I was doing here, but how come you changed to 30 if you don't mind me asking?

Reply:
oops that's a mistake. I will fix and push

Reply:
Thanks 👍